packages/ListBoxBrowser/src/components/OptionsSelected/OptionsSelected.styles.js

```diff
@@ -62,7 +61,7 @@ export const remove = css`
   display: inline-flex;
   font-size: 12px;
   height: 100%;
-  justify: flex-end;
+  justify-content: flex-end;
```

Comment:
wondering if we should left out this fix since originally was written incorrectly, so that does mean is not being applied, with the fix now its apply.

Reply:
You're right, it is not needed. I'll remove it altogether (along with another un-used style).